server/routes/auth.js

```diff
@@ -0,0 +1,37 @@
+const express = require("express");
+const router = express.Router();
+const { body } = require("express-validator");
+const User = require("../models/user");
+const authController = require("../controllers/auth-controller");
+
+router.put(
+  "/signup",
+  [
+    body("email")
+      .isEmail()
+      .withMessage("Please enter valid email")
+      .custom(async (value, { req }) => {
```

Comment:
Nit : We should rename `value` object to something meaningful

Reply:
I'll change it